### PR TITLE
Completing moving parser to own repo

### PR DESCRIPTION
**Pull request by de-sh**
As has been pointed out in the [comment](https://github.com/HTTP-APIs/hydrus/pull/340#issuecomment-466359099).

### CODE REVIEW

**Comment by de-sh** [comment]
@Mec-iS I will continue if this base seems ok to you...

**Comment by Mec-iS** [comment]
Also tests have to be ported.

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+
+install_requires = parse_requirements('requirements.txt', session=PipSession())
+dependencies = [str(package.req) for package in install_requires]
+
+setup(name='hydra_openapi_parser',
+      include_package_data=True,
+      version='0.0.1',
+      description='An OpenAPI Parser for Hydrus',
+      author='W3C HYDRA development group',
```
Comment (Mec-iS) [inline setup.py:21]: This should be "Hydra Ecosystem"

File: setup.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+      author_email='public-hydra@w3.org',
```
Comment (Mec-iS) [inline setup.py:22]: this should be "mentors@hydraecosystem.org"

**Comment by Mec-iS** [comment]
Porting as been completed with tests, if you have some changes to propose open another PR please

File: setup.py (hunk 2 above)
Comment (de-sh) [reply setup.py:22]: So is this not the format to adhere to?

Comment (xadahiya) [reply setup.py:22]: Some of the things have changed recently. We'll be using updated references from now on. Same goes for all the repos and projects.

**Pull request closed without merging**